c_tests/blockmap/Makefile

```diff
@@ -0,0 +1,3 @@
+all:
+	clang ${CFLAGS} ${LDFLAGS} -o blockmap blockmap.c
```

Comment:
Shouldn't this be:

```make
all: blockmap
    ./blockmap

blockmap: blockmap.c
    clang ${CFLAGS} ${LDFLAGS} -o blockmap blockmap.c
```

I'm not even sure we need the long command line if we set `CC` outside of the Makefile?

Reply:
That would prevent recompilation if the binary is already on disk, yeah.

> I'm not even sure we need the long command line if we set CC outside of the Makefile?

I'm not sure what you mean here though? Do you mean rely on an implicit suffix rule to do the compilation step?

Reply:
Suggest we don't worry about this, as i want to kill the `make`-based system.

That was just to get something going quickly.